samples/map/README.md

```diff
@@ -0,0 +1,5 @@
+**Map**
+
+This directory offers the following examples:
+
+  double_sequence: calculate the double of a sequence of numbers.
```

Comment:
* **double_sequence**: Given a sequence of number compute a sequence with doubles of those numbers.